src/controllers/csi/provisioner/controller.go

```diff
@@ -219,6 +219,14 @@ func (provisioner *OneAgentProvisioner) updateAgentInstallation(ctx context.Cont
 
 	latestProcessModuleConfigCache = newProcessModuleConfigCache(latestProcessModuleConfig)
 
+	if dk.NeedsOneAgentProxy() {
```

Comment:
it's a test message could you please reply on it?

Reply:
Hi andrii!

Reply:
@albertogdd one more time please

Reply:
Servus!